keylime/cloud_verifier_tornado.py

```diff
@@ -249,48 +348,61 @@ def put(self):
         try:
             rest_params = common.get_restful_params(self.request.uri)
             if rest_params is None:
-                common.echo_json_response(self, 405, "Not Implemented: Use /agents/ interface")
+                common.echo_json_response(
+                    self, 405, "Not Implemented: Use /agents/ interface")
                 return
 
             if "agents" not in rest_params:
                 common.echo_json_response(self, 400, "uri not supported")
-                logger.warning('PUT returning 400 response. uri not supported: ' + self.request.path)
+                logger.warning(
+                    'PUT returning 400 response. uri not supported: ' + self.request.path)
                 return
 
             agent_id = rest_params["agents"]
             if agent_id is None:
                 common.echo_json_response(self, 400, "uri not supported")
                 logger.warning("PUT returning 400 response. uri not supported")
-
-            agent = self.db.get_agent(agent_id)
+            try:
+                agent = session.query(VerfierMain).filter_by(agent_id=agent_id)
+            except SQLAlchemyError as e:
+                logger.error(f'SQLAlchemy Error: {e}')
 
             if agent is not None:
```

Comment:
The agent will always be `not None`. Because if it is not, then the above exception will be thrown.

Reply:
Btw, why we are throwing 404 when we find an agent? Am I missing something?

Reply:
I'm thinking this is a typo and should be 
````python 
if agent is None:
````

Reply:
ok